### PR TITLE
add training courses page and banner

### PR DESCRIPTION
**Pull request by solvingj**

### CODE REVIEW

**Review by memsharded — APPROVED** [review]
We will probably need to update the links to get CI green.

File: training_courses.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+For additional information about the Conan training series, see the original
+blog post announcement here:  
+
+- https://blog.conan.io/2020/09/22/New-conan-training-series.html
+
+For the complete list of dedicated Conan courses, see the Conan series page
+here:  
+
+- https://academy.jfrog.com/series/conan
```
Comment (uilianries) [inline training_courses.rst:22]: The academy's link is invalid, is it correct?

Comment (memsharded) [reply training_courses.rst:22]: Not published yet.

Comment (solvingj) [reply training_courses.rst:22]: also, indeed, just found out the link will be invalid.  actual link will be:
`https://academy.jfrog.com/series/jfrog-conan`
Fixing in pr

**Pull request merged**